awscli/customizations/s3/fileinfo.py

```diff
@@ -253,6 +252,60 @@ def _handle_object_params(self, params):
         if self.parameters['expires']:
             params['Expires'] = self.parameters['expires'][0]
 
+    def set_sse_request_params(self, request_params, cli_params):
```

Comment:
Why do these need to be public?  Seems like an implementation detail we don't need to expose.